Assignment/test.c

```diff
@@ -0,0 +1,110 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// test.c
+// Program to test the decimal conversion.
+// ------------------------------------------------------------------------------------------------
+#pragma warning (disable:4996)
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <limits.h> 
+#include "conversion.h"
+
+#define INVALID_INPUT "Invalid Input"
+// ANSI color code for green and red text
+#define GREEN_TEXT "\033[0;32m"
+#define RED_TEXT "\033[0;31m"
+#define RESET_TEXT "\033[0m"
+
+int IsValid (const char* n) {
+   if (n == NULL || *n == '\0') return 0;
+   char* endptr;
+   long num = strtol (n, &endptr, 10);
+   if (*endptr != '\0' || num < INT_MIN || num > INT_MAX) return 0;
+   return 1;
+}
+
+void PrintResult (const char* result) {
+   int padding = 100 - (int)strlen (result) - 1;
+   for (int i = 0; i < padding; i++) printf (" ");
+   printf ("%s\n", result);
+}
+
+void DecToBinaryTest () {
+   char* input[] = { "0", "5", "255", "-255", "2147483647", "-2147483647", "*", "abc56" };
+   char* expected_output[] = { "00000000", "00000101", "0000000011111111", "1111111100000001",

```

Comment:
```suggestion
   char* expectedOutput[] = { "00000000", "00000101", "0000000011111111", "1111111100000001",
```
Please follow camelCase while naming your variables.